<launch>
	<arg name="camera" default="bot_usb_cam" />
	
    <node name="uvc_camera" pkg="uvc_camera" type="uvc_camera_node" output="screen" ns="bot_camera/camera">
   		<param name="camera_info_url" value="file://$(find vision)/bot_usb_cam_calibration.yaml"/>
        <param name="device" value="/dev/botcam" />
        <param name="width" value="320" />
        <param name="height" value="240" />
        <param name="fps" value="30" />
        <param name="topic" type="string" value="image_raw" />
    </node>
   
</launch>
